admin/includes/modules/copy_product.php

```diff
@@ -49,7 +51,18 @@
     $contents[] = array(
         'text' => '<h6>' . TEXT_COPY_ATTRIBUTES . '</h6>' .
             '<div class="radio"><label>' . zen_draw_radio_field('copy_attributes', 'copy_attributes_yes', true) . TEXT_YES . '</label></div>' .
-            '<div class="radio"><label>' . zen_draw_radio_field('copy_attributes', 'copy_attributes_no') . TEXT_NO . '</label></div>'
+            '<div class="radio"><label>' . zen_draw_radio_field('copy_attributes', 'copy_attributes_no') . TEXT_NO . '</label></div>',
+            'params' => 'infoBoxContent duplicate-only hiddenField'
```

Comment:
i will only comment on formatting; and this file could use some work.  short array tags would be great; but it also seems that added array element with key `params` is indented wrong.  it could also benefit from a comma at the end of the value of said element.  i would say the same for line 86 below.

Reply:
You're right on the indentation.  Not sure about the array syntax since the rest of the file uses the old format.  Should I convert the whole file to use `[]`? Not sure what the zencart consensus is on this.

Reply:
On the trailing comma, it's always been my style not to use one from my early C days, but I can certainly add one if that's the style we prefer.  I assume it's for clean diffs when lines are added after or reordered.

Reply:
I have force pushed formatting fixes.  Ran the psr-12 "format document" over it, some other lines have changed as a result. Not sure what github does when you force push on a PR, I read that it will "handle it" but YMMV.